pubspec.yaml

```diff
@@ -9,4 +9,4 @@ description: >
   This library declares readable symbolic names for character codes.
 homepage: https://github.com/dart-lang/charcode
 environment:
-  sdk: '>=1.0.0 <2.0.0'
+  sdk: '>=1.0.0 <2.0.0-dev.infinity'
```

Comment:
`<2.0.0-dev.infinity` matches what we're doing with other packages. `<=3.0.0` may make sense for this one; I don't have a strong opinion here -